resources/Materials/TestSuite/stdlib/math/transform.mtlx

```diff
@@ -178,4 +178,18 @@
     </place2d>
     <output name="out" type="vector2" nodename="place2d1_1" />
   </nodegraph>
+  <!-- place2dframe -->
```

Comment:
This doesn't really show much so it might be better to add this in as a texture test so the rendered results are more obvious. For now can you please add to the check-in comment the expected output color value ? Thanks.

Reply:
The output color for place2dframe is not constant. Can I attach place2d_vector2_out_glsl.png and place2dframe_vector2_out_glsl.png in the comment?

Reply:
Or I can add a new test \resources\Materials\TestSuite\stdlib\texture\image_transform.mtlx which shows the difference between place2d & place2dframe